src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -601,7 +601,8 @@ private async Task ValidateSignUpPoliciesAsync(Guid ownerId)
             UseSecretsManager = license.UseSecretsManager,
             SmSeats = license.SmSeats,
             SmServiceAccounts = license.SmServiceAccounts,
-            LimitCollectionCreationDeletion = !flexibleCollectionsIsEnabled || license.LimitCollectionCreationDeletion
+            LimitCollectionCreationDeletion = !flexibleCollectionsIsEnabled || license.LimitCollectionCreationDeletion,
+            AllowAdminAccessToAllCollectionItems = !flexibleCollectionsIsEnabled || license.AllowAdminAccessToAllCollectionItems
```

Comment:
Same here, `AllowAdminAccessToAllCollectionItems` should use the v1 feature flag, and please rename existing variable to clearly distinguish between them.

`LimitCollectionCreationDeletion` should continue to use the mvp feature flag.